### PR TITLE
styling fixes + new icons

### PR DESCRIPTION
**Pull request by gerlofvanek**

### CODE REVIEW

**Comment by rynomster** [comment]
Doesn't seem to pass lint

File: src/app/app.component.controls.scss [hunk 1]
```diff
@@ -1,13 +1,14 @@
 :host /deep/ {
   button.control {
-    background: #03e8b0;
     border: 0;
     color: #fff;
     font-weight: 700;
     font-size: 12px;
     text-transform: uppercase;
-    padding: 10px 20px;
+	padding: 8px 30px;
```
Comment (rynomster) [inline src/app/app.component.controls.scss]: Please fix indentation

File: src/app/app.component.controls.scss [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
     vertical-align: middle;
+	background-color: #03E8B0;
```
Comment (rynomster) [inline src/app/app.component.controls.scss]: More bad indentation

File: src/app/app.component.controls.scss [hunk 3]
```diff
@@ -55,12 +56,38 @@
...
   input[type=radio][disabled] + label,
   label.disabled {
-    color: #777;
+    color: #ED4B4D;
+  }
+  
+  input[type=checkbox][disabled]:checked,
+  input[type=radio][disabled]:checked {
+    background: #ED4B4D;
+	color: #ED4B4D;
```
Comment (rynomster) [inline src/app/app.component.controls.scss]: more bad indentation

File: src/app/app.component.controls.scss [hunk 4]
```diff
@@ -55,12 +56,38 @@
...
+    appearance: none;
+    -webkit-appearance: none;
+    vertical-align: middle;
+    padding: 9px;
+    background-color: #fbfbfb;
+    border: 1px solid #ccc;
+    box-shadow: none;
+    background: #ED4B4D;
+    border-radius: 4px;
+	color: #ED4B4D;
```
Comment (rynomster) [inline src/app/app.component.controls.scss]: more bad indentation

File: src/app/app.component.controls.scss [hunk 5]
```diff
@@ -55,12 +56,38 @@
...
+	line-height: normal;
+	border-radius: 99px;
+  }
+
+  
 
   input[type=checkbox]:checked,
   input[type=radio]:checked {
     background: #03e8b0;
+	color: #03e8b0;
```
Comment (rynomster) [inline src/app/app.component.controls.scss]: more bad indentation

File: src/app/overview/overview.component.html [hunk 6]
```diff
@@ -1,24 +1,25 @@
...
     </div>
-    <div class="col-sm-6 col-md-2">
+    <div class="col-sm-6 col-md-2 padding-25">
     	<app-balance typeOfBalance="STAKE"></app-balance>
     </div>
   </div>
 </div>
 
 <div class="container-fluid">
+<div class="container-fluid">
```
Comment (rynomster) [inline src/app/overview/overview.component.html]: There are 2 divs here, one doesn't close..

File: src/app/overview/overview.component.scss [hunk 7]
```diff
@@ -9,6 +9,11 @@
 
 .balance-bg {
     background:#fff;
-    padding-bottom:25px;
+    padding-bottom:30px;
+	padding-top:20px;
```
Comment (rynomster) [inline src/app/overview/overview.component.scss]: more bad indentation

File: src/app/overview/overview.component.scss [hunk 8]
```diff
@@ -9,6 +9,11 @@
...
     border-bottom:1px solid #EAEAEA;
 }
+
+.padding-25 {
+	padding:25px;
```
Comment (rynomster) [inline src/app/overview/overview.component.scss]: please move this to style.css, where global styles are

File: src/app/settings/settings.component.html [hunk 9]
```diff
@@ -53,14 +53,16 @@
...
-          Optionnal transaction fee per kB that helps make sure your transactions
-          are processed quickly. <br /> Most transactions are 1kB. Fee 0.01 recommended.
+          <p>Optionnal transaction fee per kB that helps make sure your transactions
+          are processed quickly. <br /> Most transactions are 1kB. Fee 0.01 recommended.</p>
         </div>
         <div class="row col-xs-6">
-          <input class="col-xs-6" type="text" [(ngModel)]="settings.main.feeAmount">
-          <select class="col-xs-6" [(ngModel)]="settings.main.feeCurrency">
+          <input class="col-xs-4" type="text" [(ngModel)]="settings.main.feeAmount">
+		  <div class="col-xs-6">
```
Comment (rynomster) [inline src/app/settings/settings.component.html]: Please fix indentation - No tabs, 2 spaces

File: src/app/settings/settings.component.html [hunk 10]
```diff
@@ -74,14 +76,16 @@
...
       <div class="col-xs-8">
         <div class="input-comment row">
-          Reserved amount does not participate in staking and is therefore spendable at any time.
+          <p>Reserved amount does not participate in staking and is therefore spendable at any time.</p>
         </div>
         <div class="row col-xs-6">
-          <input class="col-xs-6" type="text" [(ngModel)]="settings.main.reserveAmount">
-          <select class="col-xs-6" [(ngModel)]="settings.main.reserveCurrency">
+          <input class="col-xs-4" type="text" [(ngModel)]="settings.main.reserveAmount">
+		   <div class="col-xs-6">
```
Comment (rynomster) [inline src/app/settings/settings.component.html]: same indentation issue

File: src/app/settings/settings.component.html [hunk 11]
```diff
@@ -140,33 +144,30 @@
...
-          <label for="version">Socket version</label>
-          <select id="version" [(ngModel)]="settings.network.socketVersion">
-            <option value="4">4</option>
-            <option value="5">5</option>
-          </select>
-        </div>
-      </div>
+      <div class="input-label col-xs-4">Proxy IP</div>
+      <input type="text" id="ip" [(ngModel)]="settings.network.proxyIP">
+	</div>
```
Comment (rynomster) [inline src/app/settings/settings.component.html]: bad indentation

File: src/app/settings/settings.component.html [hunk 12]
```diff
@@ -140,33 +144,30 @@
...
+ 
+     <div class="card row">
+      <div class="input-label col-xs-4">Port</div>
+      <input type="text" id="port" [(ngModel)]="settings.network.proxyPort">
     </div>
+	
+	<div class="card row">
```
Comment (rynomster) [inline src/app/settings/settings.component.html]: more bad indentation

File: src/app/settings/settings.component.html [hunk 13]
```diff
@@ -184,27 +185,32 @@
   <div *ngIf="tab == 'display'">
     <div class="card row">
       <div class="input-label col-xs-4">User interface language</div>
-      <select [(ngModel)]="settings.display.language">
+	  <div class="col-xs-4">
```
Comment (rynomster) [inline src/app/settings/settings.component.html]: Please fix indentation

File: src/app/settings/settings.component.html [hunk 14]
```diff
@@ -184,27 +185,32 @@
...
         <option value="part">PART</option>
         <option value="usd">USD</option>
       </select>
+	  </div>
     </div>
 
     <div class="card row">
       <div class="input-label col-xs-4">Rows per page</div>
-      <select [(ngModel)]="settings.display.rows">
+	  <div class="col-xs-4">
```
Comment (rynomster) [inline src/app/settings/settings.component.html]: same

File: src/app/wallet/history/history.component.html [hunk 15]
```diff
@@ -10,8 +10,9 @@
     <form class="header-form pull-right">
       <input type="text" placeholder="Search history">
       <button class="btn" type="submit" name="button">
-        <span class="fa fa-search"></span>
+        <span class="fa fa-search"></span>
```
Comment (rynomster) [inline src/app/wallet/history/history.component.html]: remove whitespace after >

File: src/app/wallet/receive/receive.component.html [hunk 16]
```diff
@@ -76,10 +76,10 @@
...
       </div>
       <div class="col-xs-2 row address-options vertical-center">
-        <div class="col-xs-4 icon fa fa-clone" (click)="copyAddress(address.id)"></div>
-        <div class="col-xs-4 icon fa fa-lock"></div>
-        <div class="col-xs-4 icon fa fa-qrcode" (click)="selected = address"></div>
-      </div>
+        <div class="col-xs-4 nc-icon nc-copy-2" (click)="copyAddress(address.id)"></div>
+        <div class="col-xs-4 nc-icon nc-lock"></div>
+        <div class="col-xs-4 nc-icon nc-barcode-qr" (click)="selected = address"></div>
+      </div>
```
Comment (rynomster) [inline src/app/wallet/receive/receive.component.html]: remove meaningless white space

File: src/app/wallet/receive/receive.component.scss [hunk 17]
```diff
@@ -148,30 +147,34 @@
 }
 
 button {
-  background-color: #03e8b0;
-  border: none;
+border: none;
```
Comment (rynomster) [inline src/app/wallet/receive/receive.component.scss]: fix indentation

File: src/app/wallet/send/send.component.html [hunk 18]
```diff
@@ -46,8 +46,9 @@
           [(ngModel)]="send.fromType" value="private">
         <label for="fromPrivate1" class="input-info-label">Private</label>
       </span>
-      <span class="separator"></span>
-      <span class="col-xs-4 balance">
+
+      <span class="balance">
+	    <span class="separator"></span>
```
Comment (rynomster) [inline src/app/wallet/send/send.component.html]: fix indentation

File: src/app/wallet/send/send.component.html [hunk 19]
```diff
@@ -60,12 +61,13 @@
...
         [(ngModel)]="send.toLabel">
-      <span class="separator"></span>
-      <span class="col-xs-2">
-        <span class="icon fa fa-users" (click)="openLookup()"></span>
-        <span class="icon fa fa-clone"></span>
-        <span class="icon fa fa-times"></span>
-      </span>
+
+ 
+	    <span class="separator"></span>
```
Comment (rynomster) [inline src/app/wallet/send/send.component.html]: fix indentation

File: src/app/wallet/send/send.component.html [hunk 20]
```diff
@@ -124,8 +126,8 @@
           [(ngModel)]="send.fromType"  value="private">
         <label for="fromPrivate" class="input-info-label">Private</label>
       </span>
-      <span class="separator"></span>
-      <span class="col-xs-4 balance">
+      <span class="balance">
+	    <span class="separator"></span>
```
Comment (rynomster) [inline src/app/wallet/send/send.component.html]: fix indentation

File: src/app/wallet/send/send.component.html [hunk 21]
```diff
@@ -144,8 +146,8 @@
           [(ngModel)]="send.toType" disabled="{{ send.fromType === 'private' }}" value="private">
         <label for="toPrivate" class="input-info-label">Private</label>
       </span>
-      <span class="separator"></span>
-      <span class="col-xs-4 balance">
+      <span class="balance">
+	    <span class="separator"></span>
```
Comment (rynomster) [inline src/app/wallet/send/send.component.html]: fix indentation

File: src/app/wallet/send/send.component.html [hunk 22]
```diff
@@ -157,12 +159,12 @@
...
         [(ngModel)]="send.toAddress">
       <input class="col-xs-2" type="text" placeholder="Enter a label" [(ngModel)]="send.toLabel">
-      <span class="separator"></span>
-      <span class="col-xs-2">
-        <span class="icon fa fa-users" (click)="openLookup()"></span>
-        <span class="icon fa fa-clone"></span>
-        <span class="icon fa fa-times"></span>
-      </span>
+  
+	    <span class="separator"></span>
```
Comment (rynomster) [inline src/app/wallet/send/send.component.html]: fix indentation

File: src/app/wallet/send/send.component.scss [hunk 23]
```diff
@@ -9,10 +9,15 @@
   font-size: 0.9em;
 }
 
-span {
-  padding: 0;
+.balance {
```
Comment (rynomster) [inline src/app/wallet/send/send.component.scss]: fix indentation

File: src/app/wallet/send/send.component.scss [hunk 24]
```diff
@@ -206,3 +210,8 @@ tr {
   background-color: #a1a1a1;
   border-bottom-color: #222828;
 }
+
+.padding-5
```
Comment (rynomster) [inline src/app/wallet/send/send.component.scss]: padding 5, with padding 7 top? please call it padding-top-7 or something that makes sense

File: src/index.html [hunk 25]
```diff
@@ -13,7 +13,8 @@
   	<div>
       <div class="center-index">
       	<img class="logo-index" src="assets/particl-logo-white-bg.svg">
-      	<hr />
+		      </div>
```
Comment (rynomster) [inline src/index.html]: Please fix indentation

File: src/styles.scss [hunk 26]
```diff
@@ -42,4 +42,365 @@ body {
...
+  border: 1px solid #EDEDED;
+  border-radius: 3px;
+  padding: 4px;
+  -webkit-box-shadow: none !important;
+  -moz-box-shadow: none !important;
+  box-shadow: none !important;
+  outline: none !important;
+}
+
+.form-control-settings {
```
Comment (rynomster) [inline src/styles.scss]: what's the difference between this and form-control?

File: src/styles.scss [hunk 27]
```diff
@@ -42,4 +42,365 @@ body {
...
+  padding: 4px;
+  -webkit-box-shadow: none !important;
+  -moz-box-shadow: none !important;
+  box-shadow: none !important;
+  outline: none !important;
+  margin-left: 20px;
+}
+
+.form-nomargin {
+ margin-left: 0px;
```
Comment (rynomster) [inline src/styles.scss]: please fix indentation

**Comment by rynomster** [comment]
screenshot from 2017-07-31 08-48-32
screenshot from 2017-07-31 08-48-26
screenshot from 2017-07-31 08-48-17

All the checkboxes are broken.
Please revert to working css from master.

**Comment by rynomster** [comment]
screenshot from 2017-07-31 08-49-38
screenshot from 2017-07-31 08-49-32

Pagination differs between Receive page, address book and transaction page.
We should also hide pagination if there aren't enough rows.

**Comment by gerlofvanek** [comment]
- Checkboxes are not broke. I removed the white dot and check icon, I liked it more this way (more modern and minimal) You can easy get the white dot and check icon back by changing:  
background: #ED4B4D; color: #ED4B4D; on the input @ app.component.controls.scss, and make the padding smaller. 
- Pagination are all the same, on the recieve page it's using page-buttons and not pagination. So we must add some classes and style it the same as Pagination classes. 
- We should also hide pagination if there aren't enough rows. (agreed)

**Pull request merged**